packages/mux-player/test/player.test.js

```diff
@@ -476,23 +476,22 @@ describe('seek to live behaviors', function () {
     await waitUntil(() => playerEl.inLiveWindow, 'clicking seek to live did not seek to live window');
   });
 
-  it.skip('should seek to live when play button is pressed', async function () {
+  it('should seek to live when play button is pressed', async function () {
     const playerEl = await fixture(`<mux-player
       playback-id="v69RSHhFelSm4701snP22dYz2jICy4E4FUyk02rW4gxRM"
       muted
       stream-type="ll-live"
       preload="auto"
     ></mux-player>`);
 
-    const mcPlayEl = playerEl.theme.shadowRoot.querySelector('media-play-button');
-    // NOTE: Need try catch due to bug in play+autoplay behavior (CJP)
-    try {
-      await playerEl.play();
-    } catch (_e) {}
+    await playerEl.play();
+
     await waitUntil(() => !playerEl.paused, 'play() failed');
     await waitUntil(() => playerEl.inLiveWindow, 'playback did not start inLiveWindow');
     playerEl.pause();
     await waitUntil(() => !playerEl.inLiveWindow, 'still inLiveWindow after long pause', { timeout: 7500 });
+
+    const mcPlayEl = playerEl.theme.shadowRoot.querySelector('media-play-button');
```

Comment:
it seems that if we keep this line above, the mcPlayEl and the media-play-button that's actually in the player end up being two separate elements and so the next line doesn't do anything.

Reply:
cool that makes sense. We may need to revisit the template parts architecture or how we're using it to make sure it's not unnecessarily creating entirely new DOM elements when it could just narrowly update props.